account_cost_spread/tests/test_account_cost_spread.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later [http://www.gnu.org/licenses/agpl].
+
+from openerp.tests.common import TransactionCase
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+from datetime import date
+
+
+class TestAccountCostSpread(TransactionCase):
+
+    def setUp(self):
+        super(TestAccountCostSpread, self).setUp()
+        self.acc_inv_model = self.env['account.invoice']
+        self.acc_inv_l_model = self.env['account.invoice.line']
+        self.acc_inv_spread_l_model = self.env['account.invoice.spread.line']
+        self.res_currency_model = self.env['res.currency']
+        self.partner_agrolait_id = self.env.ref(
+            "base.res_partner_2"
+        )
+        self.currency_usd_id = self.env.ref("base.USD")
+        self.account_fx_income_id = self.env.ref("account.income_fx_income")
+        self.account_fx_expense_id = self.ref("account.income_fx_expense")
+        self.product_id = self.ref("product.product_product_4")
+        self.bank_journal_usd_id = self.env.ref("account.bank_journal_usd")
+        self.account_usd_id = self.env.ref("account.usd_bnk")
+
+    def test_all(self):
+        # make product with account
+        self.product_t = self.env['product.product'].create({
+            'name': 'product_demo',
+            'type': 'service',
+            'uos_id': self.env.ref('product.product_uom_categ_unit').id,
+        })
+        # make invoice
+        self.invoice = self.acc_inv_model.create({
+            'account_id': self.account_fx_income_id.id,
+            'company_id': self.partner_agrolait_id.company_id.id,
+            'currency_id': self.currency_usd_id.id,
+            'invoice_line': [(
+                0, 0, {'account_id': self.account_fx_income_id.id,
+                       'name': 'line1',
+                       'product_id': self.product_t.id,
+                       'product_uom_qty': 2000,
```

Comment:
@gfcapalbo the field you're looking for is https://github.com/OCA/OCB/blob/8.0/addons/account/account_invoice.py#L1310